iot-hub/Quickstarts/device-streams-echo/device/DeviceClientStreamingSample.csproj

```diff
@@ -14,6 +14,6 @@
   </ItemGroup>
   
   <ItemGroup>
-	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.29.0-preview-*" />
+	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.32.0-preview-001" />
```

Comment:
This one needs to stay back because streaming isn't in the latest preview, right?

We should add an xml comment here to remind people.

Reply:
Never mind. I see 1.32.0 is lower than 1.38, and probably is the last one we shipped with streaming.

Reply:
Good point, I've already added a note in our versions tracker calling out the latest version that contains device streaming, but I'll add a note here as well.